tests/integ/test_ipinsights.py

```diff
@@ -64,6 +60,5 @@ def test_ipinsights(sagemaker_session):
         predict_input = [["user_1", "1.1.1.1"]]
         result = predictor.predict(predict_input)
 
-        assert len(result) == 1
-        for record in result:
-            assert record.label["dot_product"] is not None
+        assert len(result["predictions"]) == 1
+        assert result["predictions"][0]["dot_product"] is not None
```

Comment:
it would be better if we could test the `dot_product` value, or at least range or type instead of just not None

Reply:
Across 4 different test runs:
{'predictions': [{'dot_product': -0.22126059234142303}]}
{'predictions': [{'dot_product': -0.21824435889720917}]}
{'predictions': [{'dot_product': -0.21836185455322266}]}
{'predictions': [{'dot_product': -0.22160422801971436}]}

I added a comment to this effect in the test itself =)

Reply:
As discussed offline, added a range assertion